debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerDatabaseSchema.java

```diff
@@ -48,7 +48,7 @@ public void applySchemaChange(SchemaChangeEvent schemaChange) {
         tables().overwriteTable(table);
 
         TableChanges tableChanges = null;
-        if (schemaChange.getType() == SchemaChangeEventType.CREATE && schemaChange.isFromSnapshot()) {
+        if (schemaChange.getType() == SchemaChangeEventType.CREATE) {
```

Comment:
@jpechane, why is it that only CREATE statements are tracked?

Reply:
@gunnarmorling The fact is that from the implementaiton POV there is no difference between `CREATE` and `ALTER` so we are using a single event.

Reply:
Ok; I have to admit this still is a bit blurry to me. Would be nice to get some clarity in the context of [DBZ-993](https://issues.jboss.org/browse/DBZ-993).